substrate/frame/revive/rpc/src/client.rs

```diff
@@ -183,7 +191,7 @@ async fn extract_block_timestamp(block: &SubstrateBlock) -> Option<u64> {
 		.find_first::<crate::subxt_client::timestamp::calls::types::Set>()
 		.ok()??;
 
-	Some(ext.value.now / 1000)
+	Some(ext.value.now)
```

Comment:
was that a bug?

Reply:
agreed to change it in `pallet-revive`
BTW, this method no longer needed after building block 0 in the `pallet-revive` and let `eth-rpc` take it from storage.